docs/docs-content/clusters/pcg/manage-pcg/registry-override.md

```diff
@@ -0,0 +1,195 @@
+   | `MIRROR_REGISTRIES` | A comma-separated list of mirror registries in [image swap format](https://github.com/phenixblue/imageswap-webhook/blob/master/docs/configuration.md) to use for pulling images. For example: `docker.io::public.ecr.aws/1234567/airgap-images/docker.io,gcr.io::public.ecr.aws/1234567/airgap-images/gcr.io` | Yes      |
+
+    <details>
+    <!-- prettier-ignore -->
+    <summary>Click here for a complete example configuration.</summary>
+
+   ```yaml
+   ---
+   apiVersion: v1
+   stringData:
+     DOMAIN: "harbor.example.org"
+     BASE_PATH: "airgap-images"
+     USERNAME: ""
+     PASSWORD: ""
+     INSECURE: "false"
+     CA_CERT: ""
+     MIRROR_REGISTRIES: docker.io::harbor.example.org/airgap-images/docker.io,gcr.io::harbor.example.org/airgap-images/gcr.io,ghcr.io::harbor.example.org/airgap-images/ghcr.io,k8s.gcr.io::harbor.example.org/airgap-images/gcr.io,registry.k8s.io::harbor.example.org/airgap-images/k8s.io,quay.io::harbor.example.org/airgap-images/quay.io,us-east1-docker.pkg.dev::harbor.example.org/airgap-images
+   kind: Secret
+   metadata:
+     name: registry-info
+     namespace: jet-system
+   type: Opaque
+   ---
+   ```
+
+    </details>
+
+5. Once you have created the YAML file and configured the parameter values. Issue the following command to create the
+   Kubernetes secret containing the image registry configuration.
+
+   ```shell
+   kubectl create --filename registry-secret.yaml
+   ```
+
+## Validate
+
+Use the following steps to validate the image registry configuration.
+
+1. Open a terminal session that has network access to the PCG cluster.
+
+2. Configure kubectl to use the kubeconfig file for the PCG cluster. Use the following command to configure kubectl.
+   Refer to the [Access Cluster with CLI](../../cluster-management/palette-webctl.md) for guidance on configuring
+   kubectl.
+
+3. Issue the following command to verify that the secret containing the image registry configuration is created.
+
+   ```shell
+   kubectl get secret registry-info --namespace=jet-system  \
+   --output jsonpath='{.data.MIRROR_REGISTRIES}' | base64 --decode
+   ```
+
+   The command returns the mirror registries that you configured in the `MIRROR_REGISTRIES` parameter.
+
+   ```shell hideClipboard
+   docker.io::harbor.example.org/airgap-images/docker.io,gcr.io::harbor.example.org/airgap-images/gcr.io,ghcr.io::harbor.example.org/airgap-images/ghcr.io,k8s.gcr.io::harbor.example.org/airgap-images/gcr.io,registry.k8s.io::harbor.example.org/airgap-images/k8s.io,quay.io::harbor.example.org/airgap-images/quay.io,us-east1-docker.pkg.dev::harbor.example.org/airgap-images
+   ```
+
+4. Deploy a cluster through Palette. The PCG will propagate the image registry configuration to the workload cluster,
```

Comment:
```suggestion
4. Deploy a PCG cluster through Palette. The PCG will propagate the image registry configuration to the workload cluster,
```

Reply:
Not sure about this one, but if we mean deploying a PCG cluster, then would be good to be explicit :)